### PR TITLE
Lesson 05 classwork (strange button).

### PR DESCRIPTION
**Pull request by golovnevs**
Hello, Teachers! Here is my classwork for lesson 5 (strange button).

### CODE REVIEW

File: 05_classwork_strange_button/index_strange_button.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    <title>My ReDi Project</title>
+    <link rel="stylesheet" href="strange_button.css" type="text/css">
+</head>
+
+<body>
+    <button class="strangeButton" id="strangeButton">strange button</button>
+    <script>
+        let strangeButtonEL = document.getElementById("strangeButton");
+        let newButtonEL = document.createElement("button");
+        let remButtonEl = document.createElement("button")
```
Comment (MrTim) [inline 05_classwork_strange_button/index_strange_button.html]: Do not omit ; at the end of the statements. It's working, but it's better to have them.
Check code below for other missing ;

File: 05_classwork_strange_button/index_strange_button.html [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+            console.log('Mouse over the button');
+            strangeButtonEL.innerText = "Wow! Mouse over the button!"
+            strangeButtonEL.classList.remove('strangeButton');
+            strangeButtonEL.classList.add('mouseOver');
+            newButtonEL.innerHTML = "Genereated! Click me!";
+            document.body.appendChild(newButtonEL);
+            newButtonEL.classList.add('strangeButton');
+            document.body.appendChild(remButtonEl);
+            remButtonEl.classList.add('mouseOver');
+            remButtonEl.innerHTML = "RESET"
```
Comment (MrTim) [inline 05_classwork_strange_button/index_strange_button.html]: in some places you're using **innerText** and in some **innerHTML** for button text. Is it intentional?

File: 05_classwork_strange_button/index_strange_button.html [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        function removeButtons(event) {
+            console.log('Generated buttons removed');
+            document.body.removeChild(newButtonEL);
+            newButtonEL.classList.remove('mouseOver');
+            newButtonEL.classList.remove('clicked');
+            document.body.removeChild(remButtonEl);
+        }
+
+        function roundNewButton(event) {
+          console.log('New button clicked');
```
Comment (MrTim) [inline 05_classwork_strange_button/index_strange_button.html]: indentation in that function is different form the rest of the code.

File: 05_classwork_strange_button/index_strange_button.html [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+            strangeButtonEL.classList.add('mouseOver');
+            newButtonEL.innerHTML = "Genereated! Click me!";
+            document.body.appendChild(newButtonEL);
+            newButtonEL.classList.add('strangeButton');
+            document.body.appendChild(remButtonEl);
+            remButtonEl.classList.add('mouseOver');
+            remButtonEl.innerHTML = "RESET"
+        }
+
+        function mouseLeaveHandler(event) {
```
Comment (MrTim) [inline 05_classwork_strange_button/index_strange_button.html]: you're not using event inside your function, so you can omit it (check other places as well)

**Review by MrTim — CHANGES_REQUESTED** [review]
Nice work! I like the idea with 3 buttons and a Reset button.
You can check comments and adjust the code.

File: 05_classwork_strange_button/index_strange_button.html (hunk 1 above)
Comment (golovnevs) [reply 05_classwork_strange_button/index_strange_button.html]: Checked, fixed.

File: 05_classwork_strange_button/index_strange_button.html (hunk 2 above)
Comment (golovnevs) [reply 05_classwork_strange_button/index_strange_button.html]: No, this is mistake. Thank you! Fixed.

File: 05_classwork_strange_button/index_strange_button.html (hunk 3 above)
Comment (golovnevs) [reply 05_classwork_strange_button/index_strange_button.html]: Fixed.

File: 05_classwork_strange_button/index_strange_button.html (hunk 4 above)
Comment (golovnevs) [reply 05_classwork_strange_button/index_strange_button.html]: Fixed.

**Review by MrTim — APPROVED** [review]
Nice! Please merge this PR.

**Pull request merged**